### ISSUE TITLE
[Bug]: Games Crashes on Load on MC version 1.20.1

### ISSUE THREAD

**Issue by ModernGamingWorld** [Toadstool-Studios/Creeper-Overhaul#112]
### Bug Description

net.minecraftforge.fml.ModLoadingException: Thermal Series (thermal) encountered an error during the done event phase
§7java.lang.ClassCastException: class tech.thatgravyboat.creeperoverhaul.client.renderer.replaced.ReplacedCreeperRenderer cannot be cast to class net.minecraft.client.renderer.entity.LivingEntityRenderer (tech.thatgravyboat.creeperoverhaul.client.renderer.replaced.ReplacedCreeperRenderer is in module creeperoverhaul@3.0.1 of loader 'TRANSFORMER' @159408a7; net.minecraft.client.renderer.entity.LivingEntityRenderer is in module minecraft@1.20.1 of loader 'TRANSFORMER' @159408a7)
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:118) ~[javafmllanguage-1.20.1-47.2.17.jar%231154!/:?] {}
	at net.minecraftforge.fml.ModLoader.lambda$postEvent$29(ModLoader.java:326) ~[fmlcore-1.20.1-47.2.17.jar%231153!/:?] {re:mixin}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {re:mixin}
	at net.minecraftforge.fml.ModList.forEachModInOrder(ModList.java:227) ~[fmlcore-1.20.1-47.2.17.jar%231153!/:?] {re:mixin}
	at net.minecraftforge.fml.ModLoader.postEvent(ModLoader.java:326) ~[fmlcore-1.20.1-47.2.17.jar%231153!/:?] {re:mixin}
	at net.minecraft.client.renderer.entity.EntityRenderDispatcher.m_6213_(EntityRenderDispatcher.java:362) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:twilightforest:renderer,xf:fml:twilightforest:bake,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:twilightforest:renderer,xf:fml:twilightforest:bake,pl:mixin:APP:mixins.oculus.json:MixinEntityRenderDispatcher,pl:mixin:APP:mixins.oculus.json:entity_render_context.MixinEntityRenderDispatcher,pl:mixin:APP:mixins.oculus.compat.sodium.json:copyEntity.shadows.EntityRenderDispatcherMixin,pl:mixin:APP:embeddiumplus.mixin.json:entitydistance.MaxDistanceEntity,pl:mixin:APP:physicsmod.mixins.json:MixinEntityRenderDispatcher,pl:mixin:APP:physicsmod.mixins.json:ocean.MixinEntityRenderDispatcher,pl:mixin:APP:blueflame.mixin.json:client.MixinEntityRenderDispatcher,pl:mixin:APP:mixins.oculus.compat.sodium.json:vertex_format.entity.MixinEntityRenderDispatcher,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.packs.resources.ResourceManagerReloadListener.m_10759_(ResourceManagerReloadListener.java:15) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:computing_frames,re:mixin,re:classloading}
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?] {}
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:classloading}
	at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:115) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1106) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:redirector.mixins.json:net.minecraft.client.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:feature.measure_time.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.world_leaks.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.concurrency.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.blast_search_trees.MinecraftMixin,pl:mixin:APP:modernfix-forge.mixins.json:feature.measure_time.MinecraftMixin_Forge,pl:mixin:APP:embeddium.mixins.json:core.MinecraftClientMixin,pl:mixin:APP:botania_xplat.mixins.json:client.MinecraftAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:the_bumblezone-common.mixins.json:client.MinecraftMixin,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:craterlib.mixins.json:events.client.MinecraftMixin,pl:mixin:APP:ding.mixins.json:MinecraftMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:immediatelyfast-common.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:embeddiumplus.mixin.json:dynamiclights.MinecraftMixin,pl:mixin:APP:embeddiumplus.mixin.json:frames.FpsAccessorMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.client.AccessorMinecraft,pl:mixin:APP:carryon.mixins.json:MinecraftMixin,pl:mixin:APP:konkrete.mixin.json:MixinMinecraft,pl:mixin:APP:azurelib.forge.mixins.json:MinecraftMixin,pl:mixin:APP:travelerstitles.mixins.json:MinecraftClientTickMixin,pl:mixin:APP:fancymenu.general.mixin.json:IMixinMinecraft,pl:mixin:APP:fancymenu.general.mixin.json:MixinMinecraft,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:fallingleaves.mixins.json:MinecraftClientMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:drippyloadingscreen.mixin.json:client.IMixinMinecraft,pl:mixin:APP:drippyloadingscreen.mixin.json:client.MixinMinecraft,pl:mixin:APP:physicsmod.mixins.json:MixinMinecraft,pl:mixin:APP:physicsmod.mixins.json:cloth.MixinMinecraft,pl:mixin:APP:physicsmod.mixins.json:fabricapi.MixinMinecraft,pl:mixin:APP:mixins/common/nochatreports.mixins.json:client.MixinMinecraft,pl:mixin:APP:betterthirdperson.mixins.json:MinecraftMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:iceberg.mixins.json:MinecraftMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:securitycraft.mixins.json:camera.MinecraftMixin,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:718) ~[client-1.20.1-20230612.114412-srg.jar%231152!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:redirector.mixins.json:net.minecraft.client.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:feature.measure_time.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.world_leaks.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:bugfix.concurrency.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.blast_search_trees.MinecraftMixin,pl:mixin:APP:modernfix-forge.mixins.json:feature.measure_time.MinecraftMixin_Forge,pl:mixin:APP:embeddium.mixins.json:core.MinecraftClientMixin,pl:mixin:APP:botania_xplat.mixins.json:client.MinecraftAccessor,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:the_bumblezone-common.mixins.json:client.MinecraftMixin,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:craterlib.mixins.json:events.client.MinecraftMixin,pl:mixin:APP:ding.mixins.json:MinecraftMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:immediatelyfast-common.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:embeddiumplus.mixin.json:dynamiclights.MinecraftMixin,pl:mixin:APP:embeddiumplus.mixin.json:frames.FpsAccessorMixin,pl:mixin:APP:bookshelf.common.mixins.json:accessors.client.AccessorMinecraft,pl:mixin:APP:carryon.mixins.json:MinecraftMixin,pl:mixin:APP:konkrete.mixin.json:MixinMinecraft,pl:mixin:APP:azurelib.forge.mixins.json:MinecraftMixin,pl:mixin:APP:travelerstitles.mixins.json:MinecraftClientTickMixin,pl:mixin:APP:fancymenu.general.mixin.json:IMixinMinecraft,pl:mixin:APP:fancymenu.general.mixin.json:MixinMinecraft,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:fallingleaves.mixins.json:MinecraftClientMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:drippyloadingscreen.mixin.json:client.IMixinMinecraft,pl:mixin:APP:drippyloadingscreen.mixin.json:client.MixinMinecraft,pl:mixin:APP:physicsmod.mixins.json:MixinMinecraft,pl:mixin:APP:physicsmod.mixins.json:cloth.MixinMinecraft,pl:mixin:APP:physicsmod.mixins.json:fabricapi.MixinMinecraft,pl:mixin:APP:mixins/common/nochatreports.mixins.json:client.MixinMinecraft,pl:mixin:APP:betterthirdperson.mixins.json:MinecraftMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:iceberg.mixins.json:MinecraftMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:securitycraft.mixins.json:camera.MinecraftMixin,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:218) ~[forge-47.2.17.jar:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:flywheel.mixins.json:ClientMainMixin,pl:mixin:APP:smoothboot.mixins.json:client.MainMixin,pl:mixin:APP:physicsmod.mixins.json:MixinMain,pl:mixin:A,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {re:mixin}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.2.17.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.2.17.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.2.17.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}
Caused by: java.lang.ClassCastException: class tech.thatgravyboat.creeperoverhaul.client.renderer.replaced.ReplacedCreeperRenderer cannot be cast to class net.minecraft.client.renderer.entity.LivingEntityRenderer (tech.thatgravyboat.creeperoverhaul.client.renderer.replaced.ReplacedCreeperRenderer is in module creeperoverhaul@3.0.1 of loader 'TRANSFORMER' @159408a7; net.minecraft.client.renderer.entity.LivingEntityRenderer is in module minecraft@1.20.1 of loader 'TRANSFORMER' @159408a7)
	at net.minecraftforge.client.event.EntityRenderersEvent$AddLayers.getRenderer(EntityRenderersEvent.java:194) ~[forge-1.20.1-47.2.17-universal.jar%231157!/:?] {re:classloading,pl:eventbus:A}
	at cofh.thermal.core.client.event.TCoreClientSetupEvents.addRenderLayers(TCoreClientSetupEvents.java:32) ~[thermal_core-1.20.1-11.0.2.18.jar%231253!/:11.0.2] {re:classloading}
	at cofh.thermal.core.client.event.__TCoreClientSetupEvents_addRenderLayers_AddLayers.invoke(.dynamic) ~[thermal_core-1.20.1-11.0.2.18.jar%231253!/:11.0.2] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%2387!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2387!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2387!/:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:114) ~[javafmllanguage-1.20.1-47.2.17.jar%231154!/:?] {}
	... 31 more

### Version

creeperoverhaul-3.0.1-forge

### Mod Loader Version

1.20.1-47.2.17

### Mod Loader

Forge

### Acknowledgement

- [X] I agree that I have read what was said above this check box.

**Comment by windth001** [comment]
I went to the CoFH Team's Discord. People are reporting a conflict with Creeper Overhaul there as well. 
It was suggested to: Disable Creeper Overhaul's option to override the default creeper model or Disable the Thermal's vanilla holiday hats in the 'thermal-client' configs.

According to the persons who suggested this as a "fix" Creeper Overhaul has a client side-only functionality that is conflicting with the Thermal series creating an incompatibility.

"Creeper Overhaul replaces the creepers model with one that doesn't say its alive, and Thermal adds a hat to some mobs that say they are alive, but with the creepers model replaced the creeper says its alive but the model for the creeper says it isn't, so thermal gets confused"

Confirmed. Setting the holiday hats in the 'thermal-client' configs to false "fixed" the issue.
